### PR TITLE
Feat/open ports on anythink ack

### PR DESCRIPTION
**Pull request by darmalovan**
# Description

Change the ports visibility to public to avoid CORS issues, when the anythink services are ready.
It seems that changing the ports before the services start doesn't always work, but do work when the services are listening to a port.
Added also missing AWS image on the java folder.

### CODE REVIEW

File: .framework/java/docker-compose.yml [hunk 1]
```diff
@@ -51,9 +51,12 @@ services:
...
-    build: ./frontend
+    image: public.ecr.aws/v0a2l7y2/wilco/anythink-ack:latest
     container_name: anythink-ack
     command: sh -c "/wait-for-it.sh anythink-frontend-react:3001 -q -t 1000 && ./anythink_ack.sh"
+    environment:
+      - GITHUB_TOKEN=$GITHUB_TOKEN
+      - CODESPACE_NAME=$CODESPACE_NAME
     working_dir: /usr/src
     volumes:
       - ./:/usr/src/
```
[the remark this answers is not in the document]
Comment (darmalovan) [reply .framework/java/docker-compose.yml:62]: I can move the anythink_ack and open ports scripts to be inside the image... and then remove the volume, I can do it after if

[the remark this answers is not in the document]
Comment (NivRichter) [reply .framework/java/docker-compose.yml:62]: No biggy just a though I had. No real opinion about wether it worth it

File: .framework/java/backend/Dockerfile.aws [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM openjdk:11
+
+WORKDIR /usr/src
+COPY backend ./backend
+COPY .wilco ./.wilco
+
+# Pre-install npm packages
```
Comment (shem8) [inline .framework/java/backend/Dockerfile.aws:7]: gradle, but ok =)

**Review by shem8 — APPROVED** [review]

File: .framework/java/docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (shem8) [reply .framework/java/docker-compose.yml:62]: I think it'll just be nicer to have everything in the image, but that can wait

[the remark this answers is not in the document]
Comment (darmalovan) [reply .framework/java/docker-compose.yml:62]: I did it here - https://github.com/trywilco/wilco-engine/pull/2372 i'll change this PR accordingly

**Pull request merged**